src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -50,7 +50,34 @@
 import teammates.storage.api.StudentsDb;
 
 public class FeedbackSessionsLogic {
-
+    
+    private static final String ASSUMPTION_FAIL_DELETE_INSTRUCTOR = "Fail to delete instructor respondant for ";
+    private static final String ASSUMPTION_FAIL_RESPONSE_ORIGIN = "Client did not indicate the origin of the response(s)";
+    private static final String ERROR_NUMBER_OF_RESPONSES_EXCEEDS_RANGE = "Number of responses exceeds the limited range";
+    private static final String ERROR_SENDING_EMAILS = "Error while sending emails :";
+    private static final String ERROR_SESSION_UNPUBLISH_ALREADY = "Session is already unpublished.";
+    private static final String ERROR_SESSION_UNPUBLISH_PRIVATE = "Private session can't be unpublished.";
+    private static final String ERROR_SESSION_PUBLISH_ALREADY = "Session is already published.";
+    private static final String ERROR_SESSION_PUBLISH_PRIVATE = "Private session can't be published.";
```

Comment:
To make the error messages consistent as much as possible, maybe these can be along the lines of `"Error publishing feedback session: session is private."`, `"Error publishing feedback session: session is already published."`, `"Error unpublishing feedback session: session is already unpublished."` etc.